ironfish/src/rpc/routes/wallet/multisig/createTrustedDealerKeyPackage.ts

```diff
@@ -12,53 +15,47 @@ export type CreateTrustedDealerKeyPackageRequest = {
     identity: string
   }>
 }
+
 export type CreateTrustedDealerKeyPackageResponse = {
-  verifyingKey: string
-  proofAuthorizingKey: string
+  publicAddress: string
+  publicKeyPackage: string
   viewKey: string
   incomingViewKey: string
   outgoingViewKey: string
-  publicAddress: string
-  keyPackages: Array<{ identity: string; keyPackage: string }>
-  publicKeyPackage: string
+  proofAuthorizingKey: string
+  accounts: Array<{ identity: string; account: string }>
```

Comment:
would participants be a better name?

Reply:
Instead of `accounts`? I initially named it `participantAccounts` but then renamed it to `accounts` because the former looked too long and verbose, but I can revert it back if you think it's clearer. Naming it just `participants` is too ambiguous IMO. What do you think?